tests/Liquid/AbstractBlockTest.php

```diff
@@ -15,13 +15,6 @@
 
 class AbstractBlockTest extends TestCase
 {
-	public function testUnterminatedBlockError()
-	{
-		$this->expectException(\Liquid\Exception\ParseException::class);
-
-		$this->assertTemplateResult('', '{% block }');
-	}
```

Comment:
Is this no longer an error?

Reply:
It's not an error anymore... the new regex here fails to match that specific string and therefore is displayed unmodified.